src/Module.php

```diff
@@ -2,9 +2,7 @@
 
 namespace BsbFlysystem;
 
-use Zend\ModuleManager\Feature\ConfigProviderInterface;
```

Comment:
because of this optional line we have `zendframework/zend-modulemanager` in required. This is BC break but doesn't think someone will have a problem because of this.